apps/freenet-microblogging/Cargo.toml

```diff
@@ -14,7 +14,7 @@ panic = 'abort'
 strip = true
 
 [workspace.dependencies]
-freenet-stdlib = { version = "0.0.5", default-features = false }
+freenet-stdlib = { workspace = true, path = "../../stdlib/rust", default-features = false, features = ["contract"] }
```

Comment:
workspace = true here doesn't make sense, since this is the workspace, remove that key